app/bundles/LeadBundle/Deduplicate/ContactDeduper.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+/*
+ * @copyright   2017 Mautic Contributors. All rights reserved
```

Comment:
Sorry to be PITA, but could you check all the file annotations?

Reply:
I think I technically wrote these files in 2017 :-)